docs/_index.md

```diff
@@ -35,13 +35,14 @@ from the rest of your system.
 
 Install Poetry by downloading and executing the [installation script](https://install.python-poetry.org).
 
-**osx / linux / bashonwindows install instructions**
+**Linux, macOS, Windows (WSL)**
```

Comment:
Unfamiliar with windows, is bashonwindows the same as [WSL](https://docs.microsoft.com/en-us/windows/wsl/about)?

Reply:
Yes, LGTM :)